peekingduck/runner.py

```diff
@@ -82,9 +84,15 @@ def __init__(
             sys.exit(1)
         if RequirementChecker.n_update > 0:
             sys.exit(3)
+        if num_iter:
+            self.logger.info(f"Run pipeline for {num_iter} iterations")
+            self.num_iter = num_iter
+        else:
+            self.num_iter = 0
```

Comment:
I think specifying a negative `num_iter` breaks the code. 

Would it better to refactor this check such that only positive intergers will trigger the logging branch, and everything else will be silently defaulted to 0?
```suggestion
        if num_iter is None or num_iter <= 0:
            # Can also consider setting to None depending on
            # how you'd like to do the checks later
            self.num_iter = 0
        else:
            self.logger.info(f"Run pipeline for {num_iter} iterations")
            self.num_iter = num_iter
```

Reply:
Refactored as suggested to be clearer, and to handle negative numbers.